Makefile

```diff
@@ -2,19 +2,15 @@ default: build
 
 fmt:
 	@echo "✓ Formatting source code with gofmt..."
-	@goimports -w client
-	@goimports -w databricks
-	@goimports -w main.go
-	@gofmt -s -w client
-	@gofmt -s -w databricks
-	@gofmt -s -w main.go
+	@goimports -w $(shell find . -type f -name '*.go' -not -path "./vendor/*")
+	@gofmt -w $(shell find . -type f -name '*.go' -not -path "./vendor/*")
 	@go fmt ./...
 
 lint: 
 	@echo "✓ Linting source code with golangci-lint make sure you run make fmt ..."
 	@golangci-lint run --skip-dirs-use-default --timeout 5m
 
-test: 
+test:
```

Comment:
hm. when did i remove `lint`.

Reply:
not sure but i added lint as an explicit step in the travis builds